admin/views/redirect-delay.ejs

```diff
@@ -0,0 +1,8 @@
+<meta http-equiv="refresh" content="<%= WaitTimeBeforeRedirectInSeconds %>;url=<%= url %>" />
```

Comment:
Can you confirm that this `meta` tag will be output in the document 'HEAD'?

Reply:
Moved meta tag to head